RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.h

```diff
@@ -56,7 +56,7 @@
 #include <vector>
 
 
-class PixelThresholdClusterizer final : public PixelClusterizerBase {
+class dso_hidden PixelThresholdClusterizer final : public PixelClusterizerBase {
```

Comment:
why `dso_hidden` ?

Reply:
It's there in CMSSW `master`
https://github.com/cms-sw/cmssw/blob/148a0029c3fb3c8a92cc74b32cc5ae884bde87c5/RecoLocalTracker/SiPixelClusterizer/plugins/PixelThresholdClusterizer.h#L59

Reply:
See also the commit of #62 which (temporarily) removed it
https://github.com/cms-patatrack/cmssw/commit/766e9676854a1b785a1afe6c5800c7f3f4ec917f